src/devices/namron.ts

```diff
@@ -884,6 +884,21 @@ const definitions: Definition[] = [
         toZigbee: [],
         exposes: [e.battery(), e.temperature(), e.humidity()],
     },
+    {
+        zigbeeModel: ['4512750'],
+        model: '4512750',
+        vendor: 'Namron',
+        description: 'Namron Zigbee dimmer 2.0',
+        fromZigbee: extend.light_onoff_brightness().fromZigbee.concat(),
```

Comment:
Can you remove `fromZigbee`, `toZigbee` and `exposes` with `extend: extend.light_onoff_brightness()`?